app/controllers/users_controller.rb

```diff
@@ -43,4 +43,10 @@ def set_user
   def user_params
     params.require(:user).permit(:email, :admin)
   end
+
+  def require_admin
+    return if current_user.admin
```

Comment:
Wouldn't this return a falsy value if the user is an admin? I would have expected `return true if current_user.admin` but maybe I'm just not used to ruby

Reply:
it always returns `nil` but the redirect isn't executed because the function returns

Reply:
ah right, makes sense. it's "require admin" not "is admin"